process-automation/latest/model/data-models.md

```diff
@@ -0,0 +1,194 @@
+| Upload | *Optional.* You can upload a predefined data model. It must be in JSON schema format, for more see [JSON schema](https://json-schema.org/){:target="_blank"}. |
+| Data model description | *Optional.* A free text description of what the data model is, for example 'This data model represents a person in an organization'. |
+
+## Create a data model
+
+The following example creates a data model that represents a person, including their name, date of birth, and gender.
+
+**Note:** Data models are specific to your installation and configuration. The example described here creates a data model called Person. The model ensures that any service that uses the Person object understands its structure. This example is simple but it allows you to see how the process works but your data models could be more involved.
+
+### Create a data model for a specific process
+
+To create a data model:
+
+1. Sign into the Modeling Application and open a Project, and then click the **+** icon next to **Data Models**.
+
+    **Note:** You are able to upload already defined data models by using the **Upload** button next to Data models. Any data model you upload must be written in the JSON format.
+
+2. Enter a name for the data model, for example 'person'.
+
+3. Enter a description for the data model, for example 'This model provides the data structure for the object Person'.
+
+    You are presented with the Data Model Editor.
+
+4. From the **Type** dropdown list select **object** from the JSON schema types section.
+
+     You select object because the person data model will contain different fields.
+
+    **Note:** You can only select one JSON schema type from the **Type** dropdown list because of the rules that govern how JSON schema works. You can however select multiple other elements of the dropdown list such as aspects from the Modelling app and Composition sections. The Modelling app aspects are already predefined by the application which means you do not need to configure any fields or properties if you select them. From the Composition section you might select **anyOf** which would ensure the request contains any of the elements of Person. You can click the cog icon to access the advanced settings which are standard JSON schema configuration settings, for more see [JSON schema](https://json-schema.org/).
+
+5. Enter a meaningful title.
+
+    The title entered will be seen by a user when using a form with this data model.
+
+6. Click the **+** icon to add a property to the Person object.
+
+    A new row is created.
+
+6. Enter `Name` for the new row and then select the **Required** check box.
+
+    Selecting the required check box ensures when a new person is added, the entry must include a Name.
+
+7. From the **Type** dropdown list select **string** and then enter a meaningful title.
```

Comment:
The way it is written may lead you to think that the title is something that you add in the type dropdown